index.d.ts

```diff
@@ -1,5 +1,15 @@
 declare namespace QuickLRU {
 	interface Options<KeyType, ValueType> {
+		/**
+		The maximum number of milliseconds an item should remain in cache.
+
+		By default maxAge will be 0 which means that items will never expires.
```

Comment:
```suggestion
		By default, `maxAge` will be `0` which means that items will never expires.
```

Reply:
It's kinda weird that `0` means to never expire. I think `Infinity` would be a better default to indicate that. `0` makes it sound like it will expire right away.